index.html

```diff
@@ -1,2 +1,9 @@
 <h1>Aleksey Kabishau</h1>
-<p>Lesson 2.1: Intro to Git</p>
\ No newline at end of file
+<h3>About</h3>
+<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Sollicitudin ac orci phasellus egestas tellus rutrum. Et egestas quis ipsum suspendisse ultrices. Maecenas volutpat blandit aliquam etiam erat velit scelerisque in dictum. </p>
+<h3>Experience</h3>
+<p>Amet dictum sit amet justo donec enim. Odio ut enim blandit volutpat. Eleifend quam adipiscing vitae proin sagittis. Felis imperdiet proin fermentum leo vel orci porta. Pellentesque habitant morbi tristique senectus et netus et. Morbi blandit cursus risus at ultrices mi. </p>
+<h3>Connect</h3>
+<li>Email: aleksey.kabishau@icloud.com</li>
```

Comment:
`li` Listitems are usually wrapped in an unordered tag `ul` or an ordered tag `ol`...  would use `ul` here.

```
<ul>
    <li>Email: aleksey.kabishau@icloud.com</li>
    <li>GitHub: https://github.com/aleksey69420</li>
    <li>LinkedIn: https://www.linkedin.com/in/aleksey-kabishau/</li>
</ul>
```